app/javascript/article-form/elements/errors.jsx

```diff
@@ -6,14 +6,15 @@ const Errors = ({ errorsList }) => (
     <h2>
       <span role="img" aria-label="face screaming in fear">
         😱
-      </span>{' '}
```

Comment:
@rhymes Um, sorry about this. I removed the whitespace here but I'm assuming we still want to have some space between the emoji and "Heads up:". Shall I use the html format of the whitespace character (&nbsp;)?

Reply:
sure, we still need the space, thank you!